### ISSUE TITLE
YouTweak is abandoned

### ISSUE THREAD

**Issue by YouTweakIsBorked** [Mattie432/YouTweak#49]
New Layout on YouTube broke this extension. Author is no longer supporting this and it no longer works.

**Comment by OiYouYeahYou** [comment]
Fixing it now

**Comment by Scott216** [comment]
Yeah, that stinks.  I just uninstalled YouTweak.  I haven't been able to find anything similar. I really liked being able to click on the X to quickly hide videos.

**Comment by Nysterian** [comment]
@YouTweakIsBorked From what I see, the author is still supporting this, he labeled this as a bug 20mins ago.
I have no idea where you got that information from.

**Comment by Mattie432** [comment]
I've been inactive for a while. Currently have limited computer access. Will try to push a fix soon (ish)

**Comment by OiYouYeahYou** [comment]
I have something, and will push in a day or two

**Comment by Scott216** [comment]
That's great.  Thanks!

**Comment by inoah1** [comment]
Has this extension truly been abandoned? 
Since the last pull request on Sept 6 to fix this issue, there's been no update.

**Comment by YouTweakIsBorked** [comment]
@Nysterian
(last update 8 months ago)
yep, completely supporting it XD

**Comment by Deses** [comment]
A whole year with limited computer access...
...a dev without a computer...

(X) Doubt

**Comment by customeraccount** [comment]
> @Nysterian
> (last update 8 months ago)
> yep, completely supporting it XD

Dev said he would work irregularly on this extension.
I'm gonna keep it, and check regularly if it is fixed.
